### PR TITLE
Add relationship and minor formating

### PR DESCRIPTION
**Pull request by mattiasbarth**

### CODE REVIEW

**Review by HenrikLiman — CHANGES_REQUESTED** [review]
add ondelete='CASCADE'  on Foreign key in reviewLike, review, commentLike and comment

File: app/Data/models/comment_like.py [hunk 1]
```diff
@@ -15,7 +16,12 @@ class CommentLike(Base):
         sa.ForeignKey('comment.Id'),
         primary_key=True,
         nullable=False)
-    Liked = sa.Column(sa.Boolean, nullable=False)
+    Liked = sa.Column(
+        sa.Boolean,
+        nullable=False)
+
+    comment = relationship('Comment', back_populates='comment_like')
+    user = relationship('Comment', back_populates='comment_like')
```
Comment (AntonMaxen) [inline app/Data/models/comment_like.py]: user = relationship('Comment', back_populates='comment_like')
should be
user = relationship('User', back_populates='comment_like')

File: app/Data/models/user.py [hunk 2]
```diff
@@ -1,21 +1,55 @@
...
+        nullable=False)
+    Email = sa.Column(
+        sa.String(45),
+        nullable=False,
+        unique=True)
+    Username = sa.Column(
+        sa.String(45),
+        nullable=False,
+        unique=True)
+    Hash = sa.Column(
```
Comment (AntonMaxen) [inline app/Data/models/user.py]: missing cascades

**Review by AntonMaxen — CHANGES_REQUESTED** [review]

File: app/Data/models/comment.py [hunk 3]
```diff
@@ -19,8 +19,9 @@ class Comment(Base):
         nullable=False)
     UserId = sa.Column(
         sa.Integer,
-        sa.ForeignKey('user.Id'),
```
Comment (AntonMaxen) [inline app/Data/models/comment.py]: review cascade missing

File: app/Data/models/comment_like.py [hunk 4]
```diff
@@ -8,20 +8,20 @@ class CommentLike(Base):
...
-        sa.ForeignKey('user.Id'),
+        sa.ForeignKey('user.Id', ondelete='Cascade'),
         primary_key=True,
         nullable=False)
     CommentId = sa.Column(
         sa.Integer,
-        sa.ForeignKey('comment.Id'),
+        sa.ForeignKey('comment.Id', ondelete='Cascade'),
         primary_key=True,
         nullable=False)
```
Comment (AntonMaxen) [inline app/Data/models/comment_like.py:18]: still points to wrong class and backpopulates is wrong

File: app/Data/models/comment_like.py [hunk 5]
```diff
@@ -8,20 +8,20 @@ class CommentLike(Base):
 
     UserId = sa.Column(
         sa.Integer,
-        sa.ForeignKey('user.Id'),
+        sa.ForeignKey('user.Id', ondelete='Cascade'),
```
Comment (AntonMaxen) [inline app/Data/models/comment_like.py]: Cascade should be CASCADE no idea if it matters

**Review by AntonMaxen — CHANGES_REQUESTED** [review]

**Review by HenrikLiman — APPROVED** [review]

**Review by AntonMaxen — APPROVED** [review]

**Pull request merged**